ui/vic-ui-service/src/main/java/com/vmware/vicui/VicUIServiceImpl.java

```diff
@@ -137,8 +136,17 @@ private ServerInfo getServerInfoObject(String serverGuid) {
 		return null;
 	}
 
+    private void setThumbprint(ServerInfo sinfo) {
+        String thumbprint = sinfo.thumbprint;
+        if (thumbprint != null) {
+            _thumbprints.add(thumbprint.replaceAll(":", "").toLowerCase());
+        }
+        ThumbprintTrustManager.setThumbprints(_thumbprints);
```

Comment:
I just checked java docs on `SSLContext.getInstance` and the code and it seems `sc` will remain null if `getInstance` fails. Here are some of the docs I referenced in case you need to verify it.

http://alvinalexander.com/java/jwarehouse/openjdk-8/jdk/src/share/classes/javax/net/ssl/SSLContext.java.shtml
http://grepcode.com/file/repository.grepcode.com/java/root/jdk/openjdk/6-b14/sun/security/jca/GetInstance.java#GetInstance.getInstance%28java.lang.String%2Cjava.lang.Class%2Cjava.lang.String%29